django/src/rdwatch/urls.py

```diff
@@ -50,6 +50,10 @@
         views.satelliteimage_visual_tile,
         name='satellite-visual-tiles',
     ),
+    path(
+        'observations/<int:pk>/get-images',
```

Comment:
What do you think about using `generate-images` here instead of `get-images`? When I see `get-images`, I imagine that'd be a read-only GET endpoint.

Reply:
That's a better name.